src/Components/ImagesTable/ImagesTableToolbar.tsx

```diff
@@ -137,7 +137,7 @@ const ImagesTableToolbar: React.FC<imagesTableToolbarProps> = ({
                 '0 var(--pf-v5-c-toolbar__content--PaddingRight) 0 var(--pf-v5-c-toolbar__content--PaddingLeft)',
             }}
             isInline
-            title={`The selected blueprint is at version ${selectedBlueprintVersion}, images are at version ${latestImageVersion}. Build images to synchronize with the latest version.`}
+            title={`The selected blueprint is at version ${selectedBlueprintVersion}, some images are at version ${latestImageVersion}. Build images to synchronize with the latest version.`}
```

Comment:
Was thinking about this previously - would it make sense to say: "The selected blueprint is at version ..., _**the latest images**_ are at version .... " since we're only getting the version of the latest images?

Might be clearer than "some" images which could from the message point of view apply even if there are up to date images.

Reply:
great idea!!